includes/class-simple-local-avatars.php

```diff
@@ -343,7 +357,7 @@ public function get_simple_local_avatar_url( $id_or_email, $size ) {
 		}
 
 		// Fetch local avatar from meta and make sure it's properly set.
-		$local_avatars = get_user_meta( $user_id, $this->user_key, true );
+		$local_avatars = $this->get_user_local_avatar( $user_id );
 		if ( empty( $local_avatars['full'] ) ) {
```

Comment:
This condition might cause a PHP notice or warning if the `get_user_local_avatar()` method returns an array that doesn't have the `full` key or if an empty array is returned. To avoid the notice, we can check if the key exists before accessing it.

Reply:
Added a check in eeb66bcdf86d6ca7d455edd2f363368603a97911